reference/docs-conceptual/PowerShell-Support-Lifecycle.md

```diff
@@ -24,13 +23,30 @@ guarantee there that the community will address or resolve your issue in a timel
 have a problem that requires immediate attention, you should use the traditional, paid support
 options.
 
-## Lifecycle of PowerShell Core
+## LifeCycle of PowerShell 7
+
+With the release of PowerShell 7, support has moved from the Microsoft Modern Lifecycle Policy to follow [.NET Core's support lifecycle][Long-Term].
+In this servicing approach, customers can choose Long Term Support (LTS) releases or Current
+releases. PowerShell 7.0 is an LTS release. Support ends with the support of .NET Core 3.1.
+The next LTS release follows the next .NET core LTS release. See the [PowerShell releases end of life table](#powerShell-releases-end-of-life)
+for current ending support dates. LTS release updates only contain critical, compatible updates and
+fixes.
+
+A Current release is a release that occurs between LTS releases. Current releases can contain critical fixes, innovations, and new features. A current release is supported for three months after the next Current or LTS release.
+
+> [!IMPORTANT]
+> You must have the latest patch update installed to qualify for support. For example, if you're
+> running PowerShell 7.0 and 7.0.1 has been released, you must update to 7.0.1 to qualify for
+> support.
+
+## Lifecycle of PowerShell Core 6.x
 
-PowerShell Core is adopting the [Microsoft Modern Lifecycle Policy][modern]. This support lifecycle
+PowerShell Core used the [Microsoft Modern Lifecycle Policy][modern]. This support lifecycle
 is intended to keep customers up-to-date with the latest versions.
 
-The version 6.x branch of PowerShell Core will be updated approximately once every six months
-(examples: 6.0, 6.1, 6.2, etc.)
+The version 6.x branch of PowerShell Core was updated approximately once every six months
+(examples: 6.0, 6.1, 6.2, etc.). However, with the release of PowerShell 7, there won't be anymore
+releases of 6.x.
```

Comment:
Can you clarify this around "minor version updates"? 6.0 and 7.0 were major version updates, 6.1 and 6.2 were minor version updates, and 6.2.x will continue to receive servicing updates while it's supported.

Reply:
Updated but won't resolve unless you think it's a good representation.

Reply:
LGTM